workers_controller.js

```diff
@@ -109,6 +109,10 @@ SCWorker.create({
 								// Because of WebSocket protocol handshake restrictions, we can't call next(err) here because the
 								// error will not be passed to the client. So we can attach the error to the request and disconnect later during the SC 'handshake' event.
 								req.failedHeadersValidationError = err;
+								// When ip is blacklisted, error is thrown as soon as possible.
+								if (err.code == failureCodes.BLACKLISTED_PEER) {
```

Comment:
let's keep strict check condition with `===` check

Reply:
For consistency we should keep strict check